tutorials/algorithms/10_pvqd.ipynb

```diff
@@ -0,0 +1,591 @@
+{
```

Comment:
<p>Here I'd avoid to include any reference to the ansatz, as it doesn't have much to do with the exact calculation (it's only a variational thing)</p><p> </p><p>I'd use something like <code>initial_state = Statevector.from_label("00")</code> or at the very least stress that we are only referring to the ansatz with the initial parameters out of convenience, as we know that the state is the $|00\rangle$ (you already mentioned that fact but it can be helpful to bring that up here).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1382/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>I intentionally used the ansatz, since that code will hold for any circuit and initial point. Giving the explicit state would only work for certain states and has some additional assumptions, so as it is it's more general :) </p><p> </p>

Reply:
Yeah you have a point. The way I suggested in the previous comment would only work for basis states, so it'd be fine for the specific case of this tutorial but not in general 😅.